### ISSUE TITLE
Storage of custom keras loss function

### ISSUE THREAD

**Issue by benHeid** [KIT-IAI/pyWATTS#17]
**Describe the bug**
If pipeline.to_folder is called on a pipeline which contains a KerasWrapper with a custom loss function, then pyWATTS fails. 

**To Reproduce**
1. Build a NN with Keras with a custom loss functoin
2. Add the NN and the custom loss function to the wrapper
3. Add the wrapper to the pipline
4. Call to_folder on the pipeline

**Expected behavior**
The pipeline with the keraswrapper and the custom loss function is stored and can be reloaded correctly.
